opensearch/opensearch.gemspec

```diff
@@ -61,7 +61,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.4'
 
-  s.add_dependency 'opensearch-transport', '2.0.0'
+  s.add_dependency 'opensearch-transport', '~> 2.0.0'
```

Comment:
Curious why does the client rely on `opensearch-transport`?

Reply:
So the ruby client has 3 gems which are sub-gems kind of, `opensearch-api`, `opensearch-transport` and `opensearch-dsl`. They are all written in the client and created as separate gems plus they are used in the main gem which is the `opensearch-ruby` gem. So all the 3 gems can be used separately or can be used as part of the `opensearch-ruby` gem.